backend/dal/Migrations/Initial/PostDeploy/01-ProjectStatus.sql

```diff
@@ -65,21 +65,21 @@ INSERT INTO dbo.[ProjectStatus] (
 ), (
     6
     , 5
-    , 'Submitted'
-    , 'SU'
+    , 'Review'
+    , 'RE'
     , 1
     , 0
     , 'The project has been submitted for review to be added to the Surplus Property Program.'
-    , '/projects/submitted'
+    , '/projects/review'
     , 1
```

Comment:
Review is not a milestone.

Reply:
So wait, are you saying that this was wrong before? This change was due to PIMS-2452 which was just to rename the Submitted step to Review.

Reply:
The last step for the first workflow would be `Review`, but when they submit the last status is `Submitted`.  Which is a milestone and should start the next workflow for assessment.
There is a requirement to update the SQL scripts to reflect this, as they were an original list and there have been some changes.

There may be a issue we need to think through for the assessment workflow however.  Since `Submitted` is the first status of the assessment workflow, it may require us to handle it a little differently when presenting the data on the single form.  

We can chat through this if you want a call?